@here/harp-map-theme/resources/berlin_tilezen_day_reduced.json

```diff
@@ -1163,19 +1110,9 @@
                     "size": 14.4,
                     "opacity": 0.6,
                     "minZoomLevel": 17,
-                    "roughness": 1.95,
```

Comment:
Why removing this also?

Reply:
It's not a valid properties for a text technique. As we would make automatic validation of styles in future, we need to remove all those.

Reply:
Got it.